libs/core/pyproject.toml

```diff
@@ -20,22 +20,16 @@ dependencies = [
     "emoji==2.2.0",
     "ftfy==6.1.1",
     "kubernetes==25.3.0",
+    "langchain>=0.1.0",
     "opencc-python-reimplemented==0.1.7",
     "selectolax==0.3.17",
     "spacy==3.5.4",
     "zhipuai==1.0.7",
+    "PyMuPDF==1.24.1",
 ]
 
 [project.optional-dependencies]
 dev = ["black==23.3.0", "pylint==3.1.0"]
-experiment = [
```

Comment:
@wangxinbiao @Lanture1064 Please make sure this is approprivate

Reply:
when processing PDFs, unstructured is not used and can be deleted.